### PR TITLE
Fixed a function where a parseData was forgotten

### PR DESCRIPTION
**Pull request by AustrianDev**

### CODE REVIEW

**Review by dan-online — APPROVED** [review]
Thanks for noticing :)

**Comment by AustrianDev** [comment]
Np

File: josh-sqlite/src/index.js [hunk 1]
```diff
@@ -404,7 +404,7 @@ module.exports = class JoshProvider {
             key,
           )
         ) {
-          returnObject[key] = value;
+          returnObject[key] = this.parseData(value);
         }
       }
       lastRowId = rows.length > 0 ? rows[rows.length - 1].rowid : null;
```
Comment (extroonie) [inline josh-sqlite/src/index.js:410]:
```suggestion
      lastRowId = rows.length ? rows[rows.length - 1].rowid : null;
```
Not relevant to this PR specifically, but I think worth changing this line.

**Review by extroonie — APPROVED** [review]

File: josh-sqlite/src/index.js (hunk 1 above)
Comment (AustrianDev) [reply josh-sqlite/src/index.js:410]: I think it's not necessary, because a length can't be negative

Comment (AustrianDev) [reply josh-sqlite/src/index.js:410]: Oops, sorry i though you just added `< 0`, i will commit it

**Comment by LightBlueGamer** [comment]
https://www.npmjs.com/package/extraarrayfunctions
Wait how did this even get here, nevermind that how did i get here?!

**Pull request merged**